GOVERNANCE.md

```diff
@@ -0,0 +1,33 @@
+# Governance
+
+This document defines the project governance for OpenFunction.
+
+## Community Roles
+
+* **Users:** Members that engage with the OpenFunction community via any medium (Slack, WeChat, GitHub, mailing lists, etc.).
+* **Contributors:** Regular contributions to projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
+* **Maintainers**: The OpenFunction project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. Some Maintainers are responsible for one or more components within a project, acting as technical leads for that component. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for these components.
```

Comment:
I suggest `proactively fix bugs and perform maintenance tasks`.

Reply:
usually there is a , before and